openy.install

```diff
@@ -1004,18 +1004,6 @@ function openy_update_8068() {
   }
 }
 
-/**
- * Update Carnation theme settings: display search icon.
- */
-function openy_update_8069() {
-  if (\Drupal::service('theme_handler')->themeExists('openy_carnation')) {
-    $config = drupal_get_path('theme', 'openy_carnation') . '/config/install/openy_carnation.settings.yml';
-    /** @var Drupal\openy_upgrade_tool\ConfigParamUpgradeTool $config_importer */
-    $config_importer = \Drupal::service('openy_upgrade_tool.param_updater');
-    $config_importer->update($config, 'openy_carnation.settings', 'display_search_form');
```

Comment:
I wouldn't remove this update hook. I'd change it so that the value of the setting reflects the fact the search module is enabled.